baseframe/templates/baseframe/bootstrap3/baseframe.html.jinja2

```diff
@@ -121,6 +121,11 @@
   {%- for asset in config.get('ext_js', []) %}
     <script type="text/javascript" src="{{ asset|ext_asset_url }}"></script>
   {%- endfor %}
+
+  <!-- This block is to include JS assets of the app that are not required on all the pages but has to be included before baseframe bundle(assets "js_all").
+  For instance codemirror-markdown-js.js is only required in the formlayout pages. CodeMirror functions are called from baseframe.js, hence it has to be included before  baseframe.js -->
+  {% block appscripts %}{% endblock %}
```

Comment:
Consider calling this `pagescripts` instead of `appscripts` as it's meant for specific pages.